dotnet-dependency-injection/UsingKeyedServicesToResolveDependencies/UsingKeyedServicesToResolveDependencies/Program.cs

```diff
@@ -0,0 +1,46 @@
+using UsingKeyedServicesToResolveDependencies.Services;
+using UsingKeyedServicesToResolveDependencies.Services.Abstractions;
+
+namespace UsingKeyedServicesToResolveDependencies;
+
+public class Program
```

Comment:
We can go for the minimal appearance here as well, so you don't need program class.

Reply:
Updated.